instructure/canvas-android
# [MBL-15155][Student] K5 Grades

Test Plan:
- Verify if K5 Grades page shows the correct data (compare with web)
- Verify if the layouts are correct on Phone and Tablet (check designs)
- Verify changing grading periods shows data for correct grading period
- Enable TalkBack and check if the feature is accessible

refs: MBL-15155
affects: Student
release note: Added Grades page for K5 Homeroom
